src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -6,6 +6,7 @@
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
+import java.util.TreeSet;
```

Comment:
TreeSet is not used here. I had forgot to delete this import

Reply:
Do remember to remove it, you can update the PR through a cleanup commit.